### DISCUSSION TITLE
sync option

### DISCUSSION THREAD

**Discussion by IgnoreBox** [Authenticator-Extension/Authenticator#957] [Ideas]
based on : https://github.com/Authenticator-Extension/Authenticator/discussions/951#discussion-4401691

i think it will be nice if we have option to set auto sync with master login username? right?

**Comment by Sneezry** [comment]
There is already browser sync, is that what you are looking for?

**Reply by IgnoreBox** [reply]
aha... that's one... i miss that option on preference.

btw, how it works ? based on  google accounts or something else ? because no account  that i can (as far as i can see) to "trigger" the sync

**Reply by Sneezry** [reply]
To Chrome, it's based on Google account; to Firefox, it's based on Mozilla account; to Edge, it's based on Microsoft account.

**Reply by IgnoreBox** [reply]
i see... so it detect browser sync account ? (correct me if im wrong)

**Reply by Sneezry** [reply]
Correct. It works same as the browser's built-in syncing bookmarks, history, installed extensions, etc.

**Reply by IgnoreBox** [reply]
Thankyou for your information.... Good works....

have a nice day
